incubator-tools/advance_table_line_enhancement/ocr_walk.ipynb

```diff
@@ -0,0 +1,401 @@
+{
```

Comment:
<p>The title is not very clear. What is "OCR Walk" The description also seems similar to the other table parsing notebooks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Updated the title and description